apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -161,14 +161,22 @@ function wpcom_global_styles_enqueue_block_editor_assets() {
 		true
 	);
 	wp_set_script_translations( 'wpcom-global-styles-editor', 'full-site-editing' );
-	$plan = wpcom_site_has_global_styles_in_personal_plan() ? 'personal-bundle' : 'value_bundle';
+	$is_global_styles_in_personal_plan = wpcom_site_has_global_styles_in_personal_plan();
+	$plan                              = $is_global_styles_in_personal_plan ? 'personal-bundle' : 'value_bundle';
+
+	$reset_global_styles_support_url = 'https://wordpress.com/support/using-styles/#reset-all-styles';
+	if ( class_exists( 'WPCom_Languages' ) ) {
+		$reset_global_styles_support_url = WPCom_Languages::localize_url( $reset_global_styles_support_url );
```

Comment:
We now localize the URL in the Backend. `localizeUrl` doesn't work on ETK, so we need to do it in the Backend.